www/styles/styling.css

```diff
@@ -18,13 +18,26 @@
 
 /* -------------------------------------------------------------------------- */
 
-body {
+:root {
+	background-color: var(--clr-background);
+	color: var(--clr-text);
+}
+
+body::before {
+	content: "";
+
+	position: absolute;
+	left: 0;
+	top: 0;
+
+	height: 12rem;
+	width: 100vw;
+
 	background-image: linear-gradient(
 		185deg,
 		var(--clr-accent) 0%,
-		var(--clr-background) 12rem
+		#00000000 12rem
```

Comment:
This change is necessary because otherwise you get:

![image](https://github.com/user-attachments/assets/688b2cff-2836-4a65-8097-4a7b89ed14b2)

Note that in `#00000000` the color doesn't matter as it's entirely transparent (due to the trailing `00`).